cgt_calc/main.py

```diff
@@ -305,6 +305,10 @@ def convert_to_hmrc_transactions(
                 self.add_disposal(transaction)
                 if self.date_in_tax_year(transaction.date):
                     total_sells += self.converter.to_gbp_for(amount, transaction)
+            elif transaction.action is ActionType.GIFT:
+                raise NotImplementedError(
```

Comment:
What's the point of adding it if it errors here?

Reply:
I thought `NotImplementedError` was a more appropriate error message than the `InvalidTransactionError` that IIRC it otherwise produces (as the transaction itself is valid), so it would be easier for users to understand why the tool isn't producing calculations.